dashboard/templates/base.html

```diff
@@ -28,6 +28,19 @@
         }
     </style>
     {{ myla_globals|json_script:"myla_globals" }}
+    <script>
+        window.umConsentManager = {
+            googleAnalyticsID: "{{ myla_globals.google_analytics_id }}",
+            googleAnalyticsCustom: 
+            {
+                streamConfig: {
+                    cookie_flags: 'SameSite=None; Secure'
+                }
+            },
+            externalLinkBlank: true
```

Comment:
From what I noticed from this `externalLinkBlank` only applied to `U-M Privacy Statement` and there is another link on the banner `Privacy Policy` which is still opened from the same window and causing `Refused to frame 'https://umich.edu/' because an ancestor violates the following Content Security Policy directive: "frame-ancestors 'self'".`

Reply:
We can disable that one with

`privacyStatement: false`

But yeah, would make sense for both of them to follow this same option.